content/INTRODUCTION.md

```diff
@@ -0,0 +1,25 @@
+**Requirements**: Knowledge in PostgreSQL, PHP, and Object-Oriented Programming in PHP 
+
+**Duration**:
+
+**Standards**: K-12 Computer Science Framework Core Concepts and Practices
+* Algorithms and Programming
+    - Collaborating Around Computing
+    - Recognizing and Defining Computational Problems
+    - Developing and Using Abstractions
+    - Creating Computational Artifacts
+    - Testing and Refining Computational Artifacts
+
+## PHP & PostgreSQL
+
+PostgreSQL is an open source object-relational database system and PHP is a scripting language designed for web development. This course will cover the basics of PostgreSQL programming with PHP Data Objects(PDO). It requires `PG PHP SQL extension` for PHP and PostgreSQL to communicate.
+
+This course is designed for:
+
+* Learners who have completed `PostgreSQL Introduction`, `PHP Introduction`, `PHP Intermediate`, `PHP Advance`, and `Object-Oriented Programming in PHP`.
+
+* Individuals who have backgrounds in `PostgreSQL` and `PHP Programming`.
+
+* Experienced programmers who want to take a refresher.
+
+Get to know more about how PostgreSQL database interacts with PHP Data Objects(PDO) API. Learn how to perform database operations in PHP such as querying data, inserting data, updating data, and deleting data. Work on various exercises and problems and write your own code. Go beyond learning the syntax and experience of how PostgreSQL PHP programming works.
```

Comment:
Please try to consider this __how PostgreSQL database interacts with PHP using PHP Data Objects (PDO) API__. Thanks.

Reply:
Please add a space between `Objects` and `(PDO)`:

> Objects(PDO) ---> __Objects (PDO)__